com.nolimitconnect.NoLimitConnect.yml

```diff
@@ -0,0 +1,37 @@
+
+app-id: com.nolimitconnect.NoLimitConnect
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+only-arches: x86_64
```

Comment:
```suggestion
```

This is not supposed to be here. Please see the docs again, it should be in a flathub.json file.

Reply:
So does that mean I cannot use yml and have to use json or is this an additional file